exampleSite/config/_default/menus.en.toml

```diff
@@ -241,43 +241,55 @@
     url = "/translations"
     weight = 3
     parent = "home"
+    pre = "<i class=\"fa fa-chevron-right fa-fw\"></i>"
```

Comment:
Why were these added? If you are testing, it shouldn't be in final push, and if this is an example, it shouldn't impact existing content and should be separate.

Reply:
emmmm.... actually was hoping to have a consistency of the sidebar on home page. Do you prefer not adding it?

Reply:
removed.